### PR TITLE
Draft of homework02

### PR DESCRIPTION
**Pull request by rasi**

### CODE REVIEW

File: homeworks/homework02/homework02.Rmd [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+
+## Problem 10
+
+**5 points**
+
+Select only the bottom 10 genes with the lowest `avg_lfc` and display the
+gene `symbol`, gene `name` and `avg_lfc` for these genes.
+
+5 bonus points if you use one of the new functions you learned above!
```
Comment (k8hertweck) [inline homeworks/homework02/homework02.Rmd]: Are the new functions referenced here from Problem 1?

Comment (rasi) [reply homeworks/homework02/homework02.Rmd]: @k8hertweck Yes, I have clarified in this updated commit 5c983b2

File: homeworks/homework02/homework02.Rmd [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+```{r}
+library(tidyverse)
+```
+
+## Problem 1
+
+**10 points**
+
+Provide a <100 char description and a URL reference for each of these functions.
```
Comment (k8hertweck) [inline homeworks/homework02/homework02.Rmd]: Provide a <100 character description (in your own words)

**Comment by k8hertweck** [comment]
I made some changes to clarify what should be submitted. Let me know if you want anything to be changed!

**Comment by rasi** [comment]
Looks good. I made a minor change to display superscripts correctly. Please merge it to master once you are satisfied.

**Pull request merged**